greenkeeper.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I added this file by mistake. Remove it, please.

Reply:
ready